test_regress/t/t_timing_trace_fst.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2022 Yves Mathieu. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
```

Comment:
```suggestion
top_filename("t/t_timing_trace.v");

```

Then you can delete t_timing_trace_fst.v.

Reply:
This is what I wanted to do, but It seems that the behavior of  automatic results comparison tools differs when '--trace'  or '--trace-fst' options are involved.

- In the first case the tool waits for a ".vcd" file.
- In the second case the tool wait for a ".fst" file that is then converted to vcd for comparison with a vcd gold result.

The solution for keeping a unique Verilog file  could be to detect the kind of wanted suffix (vcd of fst) inside the Verilog source...